catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -311,6 +317,51 @@ public DeleteResponse delete(DeleteRequest deleteRequest)
     //
     // Private helper methods
     //
+    private DeleteRequest rewriteQuery(DeleteRequest deleteRequest)
```

Comment:
That sounds good. what can we use thats not too long? Only think i can think of is something like `verifyUnambigousIdentifier`